samples/powerplay/src/main/cpp/PowerPlayJNI.cpp

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <jni.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#include <android/log.h>
+
+// parselib includes
+#include <stream/MemInputStream.h>
+#include <wav/WavStreamReader.h>
+
+#include <player/OneShotSampleSource.h>
+#include "PowerPlayMultiPlayer.h"
+
+static const char *TAG = "PowerPlayJNI";
+
+#define LOG_ERROR(...) __android_log_print(ANDROID_LOG_ERROR, TAG, __VA_ARGS__)
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+using namespace iolib;
+using namespace parselib;
+using namespace oboe;
+
+// Global static player instance.
+// For more complex scenarios, consider passing this as a native peer (long) from Java.
+static PowerPlayMultiPlayer sDTPlayer;
```

Comment:
`sDTPlayer` used by DrumThumper as it uses the first letter in DrumThumper. Maybe rename this here.